### PR TITLE
status: 2023q2: service jails: minor changes

### PR DESCRIPTION
**Pull request by grahamperrin**
Markup.

Consistent use of lowercase.

Links to manual pages.

Other changes, minor.

### CODE REVIEW

File: website/content/en/status/report-2023-04-2023-06/service-jails.adoc [hunk 1]
```diff
@@ -7,8 +7,8 @@ link:https://reviews.freebsd.org/D40371[D40371: automatic service jails: some se
 
 Contact: Alexander Leidinger <netchild@FreeBSD.org>
 
-Service jails are an extension to the man:rc[8] system which allow automatic jailing of rc.d services.
-Service jails inherit the filesystem of the parent host or jail, but use all the other limits of a jail (process visibility, restricted network access, filesystem mounting permissions, sysvipc, ...) by default.
+Service jails extend the man:rc[8] system to allow automatic jailing of rc.d services.
+A service jail inherits the filesystem of the parent host or jail, but uses all other limits of the jail (process visibility, restricted network access, filesystem mounting permissions, sysvipc, ...) by default.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-04-2023-06/service-jails.adoc:11]: @netchild please, how is this for you? 

Initially, I had difficulty thinking the first sentence should be plural (Service jails … allow) or singular (an extension … allows).

**Comment by netchild** [comment]
Do the wording as you think is best (non-native speaker here). My priority is the content, not the wording.

File: website/content/en/status/report-2023-04-2023-06/service-jails.adoc [hunk 2]
```diff
@@ -1,27 +1,26 @@
-=== Service Jails - automatic jailing of rc.d services
+=== Service Jails -- automatic jailing of rc.d services
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/service-jails.adoc:1]: While here, maybe a
switch `automatic` to a capital letter.

**Review by lsalvadore — APPROVED** [review]
Approved as long as @netchild is also fine.

File: website/content/en/status/report-2023-04-2023-06/service-jails.adoc (hunk 2 above)
Comment (grahamperrin) [reply website/content/en/status/report-2023-04-2023-06/service-jails.adoc:1]: If the heading must be entirely Americanised, then (from my UK perspective) I imagine: 

> `=== Service Jails -- Automatic Jailing of rc.d Services`

**Comment by lsalvadore** [comment]
Merged, thanks.

**Pull request closed without merging**